server/src/util/adminedChapters.ts

```diff
@@ -0,0 +1,32 @@
+import { Permission } from '../../../common/permissions';
+
+interface User {
+  instance_role: {
+    instance_role_permissions: { instance_permission: { name: string } }[];
+  };
+}
+
+export const isAdminFromInstanceRole = ({ instance_role }: User) =>
+  instance_role.instance_role_permissions.some(
+    ({ instance_permission }) =>
+      instance_permission.name === Permission.ChapterEdit,
+  );
+
+export const explicitlyAdminedWhere = (user_id: number) => ({
```

Comment:
```suggestion
export const isChapterAdminWhere = (user_id: number) => ({
```
Maaaybe... what do you think?

Reply:
I like it more :rocket: